src/elm/Page/Community.elm

```diff
@@ -188,17 +188,21 @@ view loggedIn model =
 
                               else
                                 text ""
-                            , div [ class "bg-white py-6 sm:py-8 px-3 sm:px-6 rounded-lg mt-4" ]
-                                (Page.viewTitle (t "community.objectives.title_plural")
-                                    :: List.indexedMap (viewObjective loggedIn model community)
-                                        community.objectives
-                                    ++ [ if canEdit then
-                                            viewObjectiveNew loggedIn editStatus community.symbol
-
-                                         else
-                                            text ""
-                                       ]
-                                )
+                            , if community.hasObjectives then
+                                div [ class "bg-white py-6 sm:py-8 px-3 sm:px-6 rounded-lg mt-4" ]
+                                    (Page.viewTitle (t "community.objectives.title_plural")
+                                        :: List.indexedMap (viewObjective loggedIn model community)
+                                            community.objectives
+                                        ++ [ if canEdit then
+                                                viewObjectiveNew loggedIn editStatus community.symbol
+
+                                             else
+                                                text ""
+                                           ]
+                                    )
+
+                              else
+                                div [] []
```

Comment:
Actually this should be removed. Quoting the issue this PR is related:

> You'll also have to remove the "Objectives and Actions" panel, away from the community page:

Reply:
Should this be removed entirely, or only when objectives are disabled? I assume the former, but it's best to double check.

Reply:
oh we can remove it, since it will be available in the new settings page